pkg/util/watch.go

```diff
@@ -18,8 +21,26 @@ func GetWatchNamespaces() (map[string]cache.Config, error) {
 	if ns == "" {
 		return map[string]cache.Config{}, nil
 	}
+	nss := strings.Split(ns, ",")
+	nssMap := make(map[string]cache.Config)
+	for _, n := range nss {
+		nssMap[n] = cache.Config{}
+	}
+
+	return nssMap, nil
+}
 
-	return map[string]cache.Config{
-		ns: {},
-	}, nil
+// IgnoreOtherNamespaces returns the predicate for watched events that will filter out those that are not coming
+// from a watched namespace (empty namespace denotes all)
+func IgnoreOtherNamespaces() predicate.Predicate {
```

Comment:
it worked w/o this, but only when using the superman's rbac. When I was trying the per-namespace roleBindings for ScaledObjects and other CRs, the operator was failing on rbac because of this